pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java

```diff
@@ -515,6 +516,37 @@ public BundlesData getBundlesData(@PathParam("tenant") String tenant,
         return policies.bundles;
     }
 
+    @GET
+    @Path("/{tenant}/{namespace}/bundleStats")
+    @ApiOperation(value = "Get all bundles split data of this namespace.")
+    @ApiResponses(value = {
+            @ApiResponse(code = 307, message = "Current broker doesn't serve the namespace"),
+            @ApiResponse(code = 403, message = "Don't have admin permission") })
+    public List<BundleStats> getBundleStats(@PathParam("tenant") String tenant,
+                                      @PathParam("namespace") String namespace) {
+        validatePoliciesReadOnlyAccess();
+        validateNamespaceName(tenant, namespace);
+        validateNamespaceOperation(NamespaceName.get(tenant, namespace), NamespaceOperation.GET_BUNDLE);
+
+        return internalGetAllBundleStats();
+    }
+
+    @GET
+    @Path("/{tenant}/{namespace}/{bundle}/stats")
+    @ApiOperation(value = "Get the bundles split data.")
+    @ApiResponses(value = {
+            @ApiResponse(code = 307, message = "Current broker doesn't serve the namespace"),
```

Comment:
What happens if the current broker serves the namespace but doesn't host the bundle?

Reply:
I'll add lookup logic to make sure get a single bundle stats